Patch-Source: https://github.com/chimera-linux/cports/blob/867816aba3f5/main/webkitgtk/patches/cloopfix.patch

https://github.com/WebKit/WebKit/commit/c1f8a9e819201b59cacc047715b30dd2b4a1df30 was
backported to 2.42 but there's 2 fewer vars here
--
--- a/Source/JavaScriptCore/llint/LowLevelInterpreter.cpp
+++ b/Source/JavaScriptCore/llint/LowLevelInterpreter.cpp
@@ -336,8 +336,6 @@
     UNUSED_VARIABLE(t2);
     UNUSED_VARIABLE(t3);
     UNUSED_VARIABLE(t5);
-    UNUSED_VARIABLE(t6);
-    UNUSED_VARIABLE(t7);
 
     struct StackPointerScope {
         StackPointerScope(CLoopStack& stack)
